src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -47,8 +47,9 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
 
         $usesOrder = array();
         foreach ($namespacesImports as $uses) {
-            $usesOrder = array_replace($usesOrder, $this->getNewOrder(array_reverse($uses), $tokens));
+            $usesOrder[] = $this->getNewOrder(array_reverse($uses), $tokens);
         }
+        $usesOrder = call_user_func_array('array_replace', $usesOrder);
```

Comment:
Hm, I need to check if PSs' find usages will work with array_replace::class (if this valid at all), but your point is very good one here - let see if there is a better alternative